scripts/Tools/getTiming

```diff
@@ -1,669 +1,620 @@
-    prttime(' CPL:BUDGET3 '         ,$coffset,$adays,$m999,$fh);
-    prttime(' CPL:BUDGETF '         ,$coffset,$adays,$m999,$fh);
-    print $fh "\n\n";
-
-    close $fh;
-}
-
-#----------------------------------------------------------------------------------
-sub prttime{
-
-    my($str, $offset, $div,$coff, $fh) = @_;
-    my($min,$max,$found,$zoff,$maxd,$mind);
-    my($cstr,$clen,$csp);
-    
-    my $datalen = 20;
-    
-    $cstr = "<---->";
-    $clen = 6;
-    
-    gettime($str,$min,$max,$found);
-    $mind = $min;
-    $maxd = $max;
-    if ($div >= 1.0) {
-	$mind = $min/$div;
-	$maxd = $max/$div;
-    }
-    
-    my $pstrlen = 25;
-    if ($mind >= 0.0 && $maxd >= 0.0 && $found > 0.5) {
-	if ($coff >= 0) {
-	    $zoff = $pstrlen + $coff + int(($datalen-$clen)/2);
-	    $csp = $offset   - $coff - int(($datalen-$clen)/2);
-	    printf $fh " %-${zoff}s%-${csp}s %8.3f:%8.3f \n",$str,$cstr,$mind,$maxd;
-	}
-	else {
-	    $zoff = $pstrlen + $offset;
-	    printf $fh " %-${zoff}s %8.3f:%8.3f \n",$str,$mind,$maxd;
-	}
-    }
-}
-
-#----------------------------------------------------------------------------------
-sub gettime{
-
-    my($str, $min, $max, $found, $fh) = @_;
-    my(@tmp,@tmp2);
-    
-    #  set max here to something nonzero but small to avoid divide by zero
-    $found = 0;
-    $min = 0;
-    $max = 0;
-
-    my $strw = $str;
-    $strw =~ s/^\s+//;
-    $strw =~ s/\s+$//;
-    @tmp=`grep -w "$strw" $fin | grep -E '\\(' | grep -v hashtable`;
-    
-    #   print $fh "tcx1 $#tmp $tmp[0]\n";
-    if ($#tmp == 0) {
-	#       print $fh "tcx2 $tmp[0]\n";
-	#pw       if ($tmp[0] =~ m/\s*${strw}\s*\d+\s*(\d*\.\d+)\s*\(.*\)\s*(\d*\.\d+)\s*\(.*\)/) {
-	#pw2 if ($tmp[0] =~ m/\s*${strw}\s*[^\(]+\s+(\d*\.\d+)\s*\(.*\)\s*(\d*\.\d+)\s*\(.*\)/) {
-	if ($tmp[0] =~ m/\s*${strw}\s*\d+\s*\d+\s*\S+\s*\S+\s*(\d*\.\d+)\s*\(.*\)\s*(\d*\.\d+)\s*\(.*\)/) {
-	    #	   print $fh "tcxa $tmp[0]\n";
-	    #           print $fh "tcxb $1 $2 \n";
-	    $max=$1;
-	    $min=$2;
-	    $found=1.0;
-	}
-    }
-    
-    @_[1]=$min;
-    @_[2]=$max;
-    @_[3]=$found;
-    
-}
-
-#----------------------------------------------------------------------------------
-sub gettime2{
-    my($str,$procs,$count,$fin_ref)=@_;
-    my(@tmp,@tmp2);
-    
-#  initialize procs and count
-    $procs = 0;
-    $count = 0;
-    
-    my $strw = $str;
-    $strw =~ s/^\s+//;
-    $strw =~ s/\s+$//;
-    my @tmp = grep( /$strw/, @$fin_ref );
-
-    if ($#tmp == 0) {
-	#pw2 if ($tmp[0] =~ /\s*${strw}\s+([\.\de\+]+)\s+(\d+)\s/) {
-	if ($tmp[0] =~ m/\s*${strw}\s*(\d+)\s*\d+\s*(\S+)/) {
-#pw	    $procs=$2;
-#pw	    $count=$1;
-	    $procs=$1;
-	    $count=$2;
-	}
-	elsif ($tmp[0] =~ /\s*${strw}\s+(\d+)\s/) {
-	    $procs = 1;
-	    $count = $1;
-	}
-    }
-    @_[1]=$procs;
-    @_[2]=$count;
-}
-
-
+#!/usr/bin/env python
+"""
+Get timing information from run
+"""
+
+from standard_script_setup import *
+import argparse, sys, os, logging
```

Comment:
These are already imported in standard_script_setup, no need to import them again.